### PR TITLE
Bug fix: respect flags when running desktop, fix hit_proxy script

### PR DESCRIPTION
**Pull request by hwh33**

### CODE REVIEW

File: desktop/app/app.go [hunk 1]
```diff
@@ -99,6 +99,7 @@ type App struct {
 func NewApp(flags flashlight.Flags, configDir string, proClient proclient.ProClient, settings *settings.Settings) *App {
 	analyticsSession := newAnalyticsSession(settings)
 	app := &App{
+		Flags:                     flags,
```
Comment (hwh33) [inline desktop/app/app.go:102]: This was missing, causing the desktop client to completely disregard all flags.

File: hit_proxy.bash [hunk 2]
```diff
@@ -16,4 +16,10 @@ mkdir -p "$TMPDIR"
 echo "Generating config for ${PROXY} in ${OUTFILE}..."
 $LANTERN_CLOUD/bin/ptool route dump-config --legacy "$PROXY" > "$OUTFILE"
 
-make darwin ffigen && LANTERN_CONFIGDIR=$TMPDIR LANTERN_PROXYALL=true flutter run -d macOS
+make darwin ffigen
+LANTERN_CONFIGDIR=$TMPDIR \
+LANTERN_STICKYCONFIG=true \
+LANTERN_READABLECONFIG=true \
```
Comment (hwh33) [inline hit_proxy.bash:22]: We also needed these two flags.

File: desktop/app/app.go (hunk 1 above)
Comment (atavism) [reply desktop/app/app.go:102]: Great catch, thanks @hwh33

**Comment by atavism** [comment]
LGTM!

**Pull request merged**